### PR TITLE
Zach/menu bar

### PR DESCRIPTION
**Pull request by zejacobi**
Adds in a menu bar with features:
- Always visible on mobile
- Hidden until scroll past landing on desktop
- Highlights key features
- Easy to update
- No new libraries

### CODE REVIEW

**Comment by alexanian** [comment]
Bless you for doing this.

When I run this at laptop sizes, I see a hamburger menu appear in the upper right corner when I scroll down:

### Hamburger appears!
image

### Expanded (nice link styling btw)
image

I might try to expand it to a non-hamburger on mobile sometime in the lol-future, but this is v. pretty!

File: assets/sass/main.scss [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+*/
+
+// Breakpoints.
+
+	@include breakpoints((
+		default:   (1681px,   null     ),
+		xlarge:    (1281px,   1920px   ),
+		large:     (1153px,   1280px   ),
+		medium:    (737px,    1152px   ),
```
Comment (alexanian) [inline assets/sass/main.scss:22]: Okay my only confusion: what is the deal with this diff? Is this a line endings thing?

File: index.html [hunk 2]
```diff
@@ -1,123 +1,141 @@
...
+<!DOCTYPE HTML>
+<!--
+	Paradigm Shift by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<!--
+Menu adapted from: https://www.jqueryscript.net/menu/Smart-Sticky-Responsive-Navigation-With-jQuery-CSS3.html
+-->
+<html>
```
Comment (alexanian) [inline index.html:10]: Ditto this diff, like is :octocat: just having a tough night, or

**Review by alexanian — APPROVED** [review]
Alright, I am almost certainly going to merge this immediately upon waking tomorrow but I am curious what your take is on these mysterious diffs

File: index.html (hunk 2 above)
Comment (alexanian) [reply index.html:10]: (apparently there were mixed tabs and spaces in the past, for we do not live in the best of all possible worlds)

**Comment by kpande** [comment]
jfyi this user passed away last wednesday so it is unlikely to have these review questions ever answered.

**Comment by alexanian** [comment]
thanks, yeah- Zach died on August 7, and while I'd fortunately had these specific questions answered via an in-person conversation (we'd been dating since 2015) I am, uh, very acutely aware that future answers are not forthcoming

**Pull request merged**